src/components/OptionsToolbar/OptionsToolbar.js

```diff
@@ -615,10 +631,12 @@ export class OptionsToolbar extends React.Component {
       <ErrorBoundary>
         <div
           className={`${isMobile ? 'react-autoql-toolbar-mobile' : 'react-autoql-toolbar'} options-toolbar
-            ${this.state.activeMenu ? 'active' : ''}
-            ${this.props.className || ''}`}
+			${this.state.activeMenu ? 'active' : ''}
+			${this.props.className || ''}`}
           data-test='autoql-options-toolbar'
         >
+          {/* Uncomment if you want to show the Open in New button */}
```

Comment:
I think we should remove all these comments if there is no immediate plan to enable this feature, as we want to maintain a production level code base